service/templatefuncs.go

```diff
@@ -54,6 +54,13 @@ func MakeTemplateFuncs(fo TemplateFunctionOptions, superUsers []string) map[stri
 			}
 			return "America/New_York"
 		},
+		"GetUIBase": func(u gimlet.User) string {
+			usr, ok := u.(*user.DBUser)
+			if ok && usr != nil && usr.Settings.UseSpruce {
+				return "http://evergreen.spruce.s3-website-us-east-1.amazonaws.com/#"
```

Comment:
This base URL should be stored in the admin config and retrieved here (ie. not hardcoded)